debian/source/format

```diff
@@ -1 +0,0 @@
-1.0
```

Comment:
I'm *reasonably* sure you didn't mean to delete `debian/source/format`

Reply:
Thank you for being so understanding